### PR TITLE
Added option to include the DEM file per mintPy developer request

### PR DESCRIPTION
**Pull request by talogan**

### CODE REVIEW

**Comment by asjohnston-asf** [comment]
Possibly out of scope for this PR, but I notice the DEM geotiff comes out with a pixel size of 80m/40m for 20x4/10x2 looks, even though we originally downsample the input DEM to 160m/80m.  I looks like that's because we set oversampling to `2` when `GC_map_mod` and `gc_map`, which resamples the DEM back to 80m/40m:
https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/SLC_copy_S1_fullSW.py#L63

That seems unproductive?  Should we resample the input DEM to 80m/40m at the start and avoid possibly throwing away resolution?

File: hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt [hunk 1]
```diff
@@ -144,6 +145,13 @@ The water mask specifies, pixel-by-pixel, where there is land and where there is
 
 *Note that this file is optional. Select the "Include Water Mask" option in the HyP3 Processing Options to include it in the product package.*
 
+-------------
+## 11. Digital Elevation Model
+
+The digital elevation file contains the terrain height for the scenes being processed. The values are taken from the Copernicus GLO-30 Public DEM.
```
Comment (asjohnston-asf) [inline hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt:151]: We've proposed adding "The height values will differ from the original Copernicus DEM dataset, as a geoid correction has been applied" to the product guide.  I'd lean towards adding that note here, as well.

**Comment by asjohnston-asf** [comment]
Do you intend to update the changelog and add the new option to the HyP3 insar entrypoint at https://github.com/ASFHyP3/hyp3-gamma/blob/adding_dem_file/hyp3_gamma/__main__.py#L87?  Or should I grab those?

File: hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt (hunk 1 above)
Comment (talogan) [reply hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.5.0]
+
+## Added
```
Comment (asjohnston-asf) [inline CHANGELOG.md:11]: should be H3

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+* Option to include DEM file with InSAR products
+  * `include_dem` parameter to ifm_sentinel.insar_sentinel_gamma` function
+  * `--include-dem` parameter to `insar` entrypoint
+  * `-d` parameter to `ifm_sentinel.py` script
```
Comment (asjohnston-asf) [inline CHANGELOG.md:15]: missing blank line here

File: hyp3_gamma/insar/ifm_sentinel.py [hunk 4]
```diff
@@ -265,7 +270,8 @@ def make_parameter_file(mydir, parameter_file_name, alooks, rlooks, dem_source):
 
 
 def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, include_look_vectors=False,
-                         include_los_displacement=False, include_wrapped_phase=False, include_inc_map=False):
+                         include_los_displacement=False, include_wrapped_phase=False, include_inc_map=False,
+                         include_dem=False):
```
Comment (asjohnston-asf) [inline hyp3_gamma/insar/ifm_sentinel.py:274]: changelog calls out "--include-dem` parameter to `insar` entrypoint", should probably do that as part of this PR

File: hyp3_gamma/__main__.py [hunk 5]
```diff
@@ -90,6 +90,7 @@ def insar():
     parser.add_argument('--password', required=True)
     parser.add_argument('--bucket')
     parser.add_argument('--bucket-prefix', default='')
+    parser.add_argument('--inlucde-dem', type=string_is_true, default=False)
```
Comment (asjohnston-asf) [inline hyp3_gamma/__main__.py:93]: `include_dem` typo

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**